addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -162,10 +158,13 @@
                             </a>
                         </t>
                     </t>
+                </td>
+                <td valign="middle">
```

Comment:
What's the difference VS vertical-align: middle in css?

Looks like this property is not supported in HTML5 (not that it matters too much for emails though).

EDIT ok I see valign existing here below, nevermind then I guess you're just re-using it.